k8s/helm-chart/dynatrace-gcp-monitor/values.yaml

```diff
@@ -124,9 +124,162 @@ metricResources:
     cpu: "1.25"
     ephemeral-storage: "2Gi"
 # metricAutodiscovery: If true additional metrics not listed in extensions will be automatically detected and monitored
-metricAutodiscovery: "true"
+metricAutodiscovery: "false"
 # Metrics autodiscovery polling interval in minutes.
 autodiscoveryQueryInterval: 60
+# configuration file for autodiscovery
+autodiscoveryResourcesYaml: |
```

Comment:
Could we change the approach of configuring the desired autodiscovered services. Customer can put a simple comma-separated list (or something like that) of services directly, assuming he knows them from Google (not to overload the values.yaml).

Reply:
moved to autodiscovery-values.yaml